src/redis_cmd.cc

```diff
@@ -4388,7 +4241,11 @@ Status LookupCommand(const std::string &cmd_name,
   if (cmd_factory == commands.end()) {
     return Status(Status::RedisUnknownCmd);
   }
-  *cmd = cmd_factory->second.factory();
+  auto redisCmd = cmd_factory->second;
```

Comment:
`cmd_factory` is not a fine variable name currently.

Reply:
good catch! any advice? how about cmd_iter

Reply:
`cmd_iter` is ok to me
How about changing `LookupCommand` to `LookupAndCreateCommander`